boxsdk/auth/__init__.py

```diff
@@ -7,7 +7,14 @@
 try:
     from .jwt_auth import JWTAuth
 except ImportError:
-    JWTAuth = None  # If extras are not installed, JWTAuth won't be available.
+    JWTAuth = None  # If extras[jwt] are not installed, JWTAuth won't be available.
 from .oauth2 import OAuth2
-from .redis_managed_oauth2 import RedisManagedJWTAuth, RedisManagedOAuth2
+try:
+    from .redis_managed_oauth2 import RedisManagedOAuth2
+except ImportError:
+    RedisManagedOAuth2 = None  # If extras[redis] are not installed, JWTAuth won't be available.
```

Comment:
Or maybe not. I guess the comment is correct, but a bit confusing since JWTAuth isn't what you're trying to import.